src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  let tempState = state;
+
+  for (const singleAction of actions) {
```

Comment:
I think that you could simply name it action
```suggestion
  for (const action of actions) {
```